Bottom: 48f138c25d9e32aa9c929d1933a58278fc39a9de
Top:    4c4433d906639aed55de7876e8d857d10601b643
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 23:21:28 +0100

Refresh of consolidate-pin-drawing-routin

---

diff --git a/src/crosshair.c b/src/crosshair.c
index cd52f4a..f016094 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -249,7 +249,7 @@ XORDrawElement (ElementTypePtr Element, LocationType DX, LocationType DY)
     PinType moved_pin = *pin;
     moved_pin.X += DX; moved_pin.Y += DY;
 
-    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pin, true, false);
+    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pin, false, false);
   }
   END_LOOP;
